app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] You forgot to set up the associations between `user and likes, comment, post` you should do that for example the post can have many likes, and comments, and the user belongs to the post  so it should the code like this :
```RUBY
belongs_to :author, class_name: 'User'
```